### PR TITLE
Add support for pyenv-virtualenvwrapper

### PR DESCRIPTION
**Pull request by ClassicOldSong**

### CODE REVIEW

File: plugins/available/virtualenv.plugin.bash [hunk 1]
```diff
@@ -3,7 +3,11 @@
 cite about-plugin
 about-plugin 'virtualenvwrapper helper functions'
 
-[[ `which virtualenvwrapper.sh` ]] && . virtualenvwrapper.sh
+if [[ `command -v pyenv` ]]; then
```
Comment (nwinkler) [inline plugins/available/virtualenv.plugin.bash]: You should be able to use the global `_command_exists` function here:

```bash
if _command_exists pyenv; then
```

See https://github.com/Bash-it/bash-it/blob/a510cd0fd35fd7050ea24a09c2d08f0c2073e1a8/lib/helpers.bash#L8-L15 for details

File: plugins/available/virtualenv.plugin.bash [hunk 2]
```diff
@@ -3,7 +3,11 @@
 cite about-plugin
 about-plugin 'virtualenvwrapper helper functions'
```
Comment (nwinkler) [inline plugins/available/virtualenv.plugin.bash]: You might want to add the support for pyenv here, too.

**Comment by ClassicOldSong** [comment]
@nwinkler done

**Comment by nwinkler** [comment]
Great - thanks!

**Pull request merged**